### PR TITLE
wallet documentation

### PR DESCRIPTION
**Pull request by akekeke**
This PR
- adds README.md to wallet package

### CODE REVIEW

File: internal/wallet/utxo.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/btcsuite/btcwallet/wtxmgr"
 )
 
-// Utxo is a unspend transaction output
+// Utxo is a unspent transaction output
```
Comment (Jwata) [inline internal/wallet/utxo.go]: 🙇

File: internal/wallet/READMe.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# wallet
```
Comment (Jwata) [inline internal/wallet/READMe.md]: READMe.md -> README.md

File: internal/wallet/READMe.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+Under each `ScopedKeyManager` are `Account` types associated with that `ScopedKeyManager`.
+
+For more information on address management, please consult the original [godoc](https://godoc.org/github.com/btcsuite/btcwallet/waddrmgr).
+
+## How UTXO management works
+
+Right now, to ask about UTXOS, the wallet will query the running `bitcoind` instance by using the rpc command `ListUnspent()`.
+When a public address is generated, it is also registered to `bitcoind`, so `bitcoind` knows to keep track of transactions associated with that addresss.
+
+## Connecting to `bitcoind`
```
Comment (Jwata) [inline internal/wallet/READMe.md]: This is more like TODO. So I would move to comment in the wallet package.

Comment (akekeke) [reply internal/wallet/READMe.md]: I think its good to include this section here because it's important for using the `wallet` package.

**Pull request merged**